### ISSUE TITLE
build_targets Index Out Of Bounds Error

### ISSUE THREAD

**Issue by ganLover** [Ray-Luo/YOLOV3-PyTorch#3]
While training the system on custom dataset, [x_center, y_center, width, height] (format of the ground truth- wrt. the image) eg- (93,100,60,70)

In the buildTargets():

           gx = targets[batch_idx,target_idx, 1] * grid_size
            gy = targets[batch_idx,target_idx, 2] * grid_size
            gw = targets[batch_idx,target_idx, 3] * grid_size
            gh = targets[batch_idx,target_idx, 4] * grid_size

            # get grid box indices
            gi = int(gx)
            gj = int(gy)
.......

conf_mask[batch_idx, anchor_iou > ignore_thres, gj, gi] = 0

As per example ->
gx = 93 * 13
gi = 93 * 13

But -> conf_mask = torch.ones(batch_size,num_anchors,grid_size,grid_size)
                                                [Batch_size, no_of_anchors, grid_size, grid_size]
Where grid_size = 13

That is why I am getting index out of bound error.
If I am wrong pls explain what is the format of Target labels.
